index.html

```diff
@@ -0,0 +1,184 @@
+    <a href="index.html" class="logo"><img src="source/images/logo.svg" alt="GOLD GYM logo"></a>
+    <!-- fa-solid fa-x -->
+    <button id="show-menu"><i class="fas fa-bars" id="change-icon"></i></button>
+    <!-- nav menu -->
+    <ul class="nav-menu" id="nav-menu">
+      <li class="nav-item"><a href="index.html" class="nav-link" id="link-active">Home</a></li>
+      <li class="nav-item"><a href="about-us.html" class="nav-link">About</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+    </ul>
+  </nav>
+
+  <!-- Hero Section -->
+  <header class="hero-wrapper">
+    <div class="hero-container">
+      <p class="greeting">"Hello Athlete's"</p>
+      <h1 class="hero-header-text">
+        Victory isn't defined by wins and losses, it's defined by effort
+      </h1>
+      <p class="hero-header-desc">With our team you’ll reach your goals quickly. Enjoy a spacious and well equipped free
+        weight area to meet everyone's needs with the best equipment in the industry. Equipped with the space and
+        equipment for the normal person to competitive athletes or athletic teams.
+      </p>
+      <p class="hero-header-time"><span class="workout-time"><i class="fas fa-clock"></i>24/7</span> (Monday) ~ (Friday)
+      </p>
+    </div>
+  </header>
+
+  <!-- Services Section -->
+  <section class="services">
+    <h1 class="services-header">Main Services</h1>
+    <div class="card-wrapper">
+      <!-- Card-1 -->
+      <div class="card">
+        <div class="card-image">
+          <i class="fas fa-duotone fa-dumbbell"></i>
+        </div>
+        <div class="card-header">
+          Body Building
+        </div>
+        <div class="card-body">
+          <p>Build your body with great instactore in the industry. With complete Nutration plane.</p>
+        </div>
+      </div>
+      <!-- Card-2 -->
+      <div class="card">
+        <div class="card-image">
+          <i class="fas fa-clock"></i>
+        </div>
+        <div class="card-header">
+          OPEN 7 DAYS A WEEK
+        </div>
+        <div class="card-body">
+          <p>Flexible hours to meet everyone's scheduling needs.</p>
+        </div>
+      </div>
+      <!-- Card-3 -->
+      <div class="card">
+        <div class="card-image">
+          <i class="fa-sharp fa-solid fa-hand-sparkles"></i>
+        </div>
+        <div class="card-header">
+          CLEAN, SPACIOUS FACILITIE
+        </div>
+        <div class="card-body">
+          <p>From front lobby and fitness machines to the bathrooms , a clean gym is one of our top priorities.</p>
+        </div>
+      </div>
+    </div>
+    <a href="#" class="btn-join">Join GOLD GYM</a>
+  </section>
+
+  <!-- Feature Speaker -->
+  <section class="services feature-speaker">
+    <h1 class="services-header speaker-header">Clients Feedback</h1>
```

Comment:
- [ ] There should be no more than 1 `<h1>` tag on a page. Please take a look at [this article](https://www.contentkingapp.com/academy/headings/) about the headings and fix this issue.